wordpress/wp-content/plugins/cds-wpml-mods/resources/js/util/fetch.js

```diff
@@ -21,14 +21,14 @@ export const getData = async (endpoint) => {
     return await response.json();
 };
 
-export const sendData = async (endpoint, data) => {
+export const sendData = async ({ endpoint, data, method }) => {
     const requestHeaders = new Headers({
         'Content-Type': 'application/json;charset=UTF-8',
     });
     requestHeaders.append('X-WP-Nonce', CDS_VARS.rest_nonce);
 
     const response = await fetch(`${CDS_VARS.rest_url}${endpoint}`, {
-        method: 'POST',
+        method: method,
```

Comment:
We should validate the method here so an arbitrary value isn't passed.

Reply:
Makes sense, updated.